copier/main.py

```diff
@@ -211,14 +211,12 @@ def __enter__(self) -> Worker:
         return self
 
     @overload
-    def __exit__(self, type: None, value: None, traceback: None) -> None:
-        ...
+    def __exit__(self, type: None, value: None, traceback: None) -> None: ...
 
     @overload
     def __exit__(
         self, type: type[BaseException], value: BaseException, traceback: TracebackType
-    ) -> None:
-        ...
+    ) -> None: ...
```

Comment:
Could you please reformat these lines in the first commit, so none of the two commits contain any formatting changes here?